app/src/pages/Register.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react"
```

Comment:
unclear on what this component is @martyboggs @tylerthome - do we need this or can we delete it?

Reply:
Oh yeah, I built out some routes to get an idea of the flow with organizations and host/guest. I'll delete this now.